code/index.html

```diff
@@ -1,21 +1,367 @@
+              <span class="material-icons-round white icon"> nightlife </span>
+            </div>
+            <div class="ablauf-text">
+              <h3>18:00 Uhr</h3>
+              <p class="schedule-text">Unser Hochzeitsfest</p>
+            </div>
+            <div class="map">
+              <iframe
+                src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d1349.4999012920734!2d9.2418809!3d47.4314472!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x479ae6d5ea26baaf%3A0x8e5e8f756d4a3a5d!2sHofstadl!5e0!3m2!1sde!2sch!4v1696101870503!5m2!1sde!2sch"
+                width="600"
+                height="450"
+                style="border: 0"
+                allowfullscreen=""
+                loading="lazy"
+                referrerpolicy="no-referrer-when-downgrade"
+              ></iframe>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <div id="Anmeldung">
+      <h2>Anmeldung</h2>
+      <p>
+        Bitte meldet euch bis am <strong>31. Januar 2024</strong> mit folgendem
+        Formular an oder auch ab, damit wir planen können.
+      </p>
+    </div>
+    <div id="Trauzeugen">
+      <h2>Unsere Trauzeugen</h2>
+      <ul class="row">
+        <li class="column-50">
+          <div class="Trauzeuge-left">
+            <img
+              class="round-image"
+              src="./images/Abendessen.jpg"
+              alt="Trauzeugin Lea"
+            />
+            <h3>Simea</h3>
+            <p>Simea und ich kennen uns blablaba</p>
+            <a href="mailto:sabrina.lombriser@gmail.com">Mail</a><br />
+            <a href="tel:0041793559220">079'355'92'20</a>
+          </div>
+          <div class="Trauzeuge-left">
+            <img
+              class="round-image"
+              src="./images/Bier.jpg"
+              alt="Trauzeuge Marc"
+            />
+            <h3>Ardillo</h3>
+            <p>ominöser Trauzeuge von Marc</p>
+            <a href="mailto:sabrina.lombriser@gmail.com">Mail</a><br />
+            <a href="tel:0041793559220">079'355'92'20</a>
+          </div>
+        </li>
+      </ul>
+    </div>
+    <div id="Wunschliste">
+      <h2>Unsere Wunschliste für unsere Hochzeit</h2>
+      <p>
+        Bist du kreativ? Dann lassen wir uns gerne davon überraschen.<br />
+        Ansonsten wisst ihr vielleicht, dass wir unsere Hochzeitsreise im Norden
+        verbringen wollen. Hierfür benötigen wir noch ein wenig Unterstützung.
+      </p>
+      <ul class="Wunschliste">
+        <li class="wishlist">
+          <div class="wunsch-container">
+            <div class="wunsch-Bild">
+              <img
+                class="image-wunsch"
+                src="./images/Abendessen.jpg"
+                alt="Bild zNacht"
+              />
+            </div>
+            <div class="wunsch-Text">
+              <p>
+                Gemütliches Nachtessen <br />
+                <strong>100 CHF</strong>
+              </p>
+            </div>
+          </div>
+        </li>
```

Comment:
You could simplify the html structure for all the `li` elements as follows (with a few CSS changes; see other comment):
```suggestion
        <li class="wishlist wunsch-container">
              <img
                class="image-wunsch"
                src="./images/Abendessen.jpg"
                alt="Bild zNacht"
              />
              <p class="wunsch-Text">
                Gemütliches Nachtessen <br />
                <strong>100 CHF</strong>
              </p>
        </li>
```